fili-navi/src/test/java/com/yahoo/bard/webservice/data/dimension/TestTaggedDimensionField.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.dimension;
+
+import static com.yahoo.bard.webservice.data.dimension.impl.DefaultDimensionFieldTag.PRIMARY_KEY;
+
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Test implementation of TaggedDimensionField used in testings.
+ */
+public enum TestTaggedDimensionField implements TaggedDimensionField {
+    TEST_PRIMARY_KEY(Collections.singletonList(PRIMARY_KEY)),
+    TEST_DISPLAY_NAME(Arrays.asList(PRIMARY_KEY, PRIMARY_KEY)),
+    TEST_DESCRIPTION(Collections.<Tag>emptyList())
+    ;
```

Comment:
If these fields exist only to illustrate behavior on single tag, multiple tags and no tags, you might want to just rename the dimensions to reflect that purpose.  

Also, would it be clearer to instantiate these tags as singletons in the test class?  That would make it easer to see what you're testing by putting it next to the test itself, if the test is the only place these are being consumed.